spec/Http/Guzzle/ClientAdapterSpec.php

```diff
@@ -2,18 +2,17 @@
 
 namespace spec\Cardinity\Http\Guzzle;
 
-use Cardinity\Http\ClientInterface;
 use Cardinity\Http\Guzzle\ExceptionMapper;
 use Cardinity\Method\MethodInterface;
-use GuzzleHttp\Client;
+use GuzzleHttp\ClientInterface;
 use GuzzleHttp\Exception\ClientException;
 use Psr\Http\Message\ResponseInterface;
 use PhpSpec\ObjectBehavior;
 
 class ClientAdapterSpec extends ObjectBehavior
```

Comment:
It looked to me like we were testing GuzzleClient with our interface, but the spec is for our client so it was changed.